packages/lib-user/src/components/GroupStats/components/TopContributors/TopContributors.js

```diff
@@ -42,36 +45,62 @@ function TopContributors({
         />
       }
     >
-      <Grid
-        as='ol'
-        columns={size === 'small' ? ['1fr'] : [ '1fr', '1fr' ]}
-        gap='small'
-        pad='none'
-        rows={['repeat(5, auto)']}
-        style={{
-          gridAutoFlow,
-          listStyle: 'none'
-        }}
-      >
-        {topContributorsWithStats?.length ? (
-          topContributorsWithStats.map((user) => (
-            <li key={`MemberCard-${user?.id}`}>
-              <MemberCard
-                avatar={user?.avatar_src}
-                classifications={user?.classifications}
-                displayName={user?.display_name}
-                login={user?.login}
-              />
-            </li>
-          ))
-        ) : null}
-      </Grid>
+      {loading ? (
+        <Box
+          align='center'
+          fill
+          justify='center'
+          pad='medium'
+        >
+          <Loader />
+        </Box>
+      ) : error ? (
```

Comment:
Idk why I never considered it before, but chaining ternary operators is a really nice way to handle UI states here 👍 I might have to refactor a few on the homepage.